content/docs/command-reference/pull.md

```diff
@@ -217,8 +217,16 @@ $ dvc remote list
 r1	ssh://_username_@_host_/path/to/dvc/remote/storage
 ```
 
+<<<<<<< HEAD With the first `dvc pull` we specified a stage in the middle of
+this pipeline (`matrix-train.p.dvc`) while using `--with-deps`. DVC started with
+that `.dvc` file and searched backwards through the pipeline for data files to
+download. Because the `model.p.dvc` stage occurs later, its data was not pulled.
+=======
+
 > DVC supports several
 > [remote types](/doc/command-reference/remote/add#supported-storage-types).
+>
+> > > > > > > c8b720a9017ca9db2caae7fc9f521f6192fc4f4c
```

Comment:
@utkarshsingh99 unresolved merge conflict here. Please be very careful never to commit these to git. Please resolve and check whether there are any others in this PR or any other of your branches. (Just grep for `<<<<<<<`.)

p.s. running the local dev app would help you detect these, as of course they break the app. Please always run it and review your work in localhost before pushing. Thanks

Reply:
Oh, okay. I'll keep that in mind.
What should I do with this conflict?
Accept both of the texts?
Since the HEAD change was your addition and the incoming change is @imhardikj 's addition from 12 days ago.

Reply:
> What should I do with this conflict?
> Accept both of the texts?

You usually can't just pick one OR the other. Solving conflicts means bits of both are needed in the final code.

Reply:
Let me check...

Reply:
Yeah it was the lower one 😋  Anyway, I fixed this. Resolving.